Makefile

```diff
@@ -29,3 +30,11 @@ SHELL_OPTS= -name ${PROJECT}@`hostname` -s sync
 
 test-shell: build-ct-suites app
 	erl -pa ebin -pa deps/*/ebin -pa test -s sync -s lager
+
+# Builds the xref_runner escript.
+
+escript: all
+	rebar escriptize
```

Comment:
rebar should be a macro parameter, like `${REBAR}` so that the user can define it's own. To do that, you need to add (at the beginning of the Makefile) something like:
```makefile
REBAR?=rebar
```